src/test/java/Helper/DateUtil.java

```diff
@@ -2,12 +2,42 @@
 
 import java.text.ParseException;
 import java.text.SimpleDateFormat;
-import java.util.Arrays;
+import java.util.ArrayList;
 import java.util.Calendar;
 import java.util.Date;
+import java.util.List;
 
 public class DateUtil {
 
+    public static Date getCurrentDate(){
+        Date currentDate=new Date();
```

Comment:
Instead of creating local object of Date 4 times inside 4 different methods, you can create field variable and use it inside all the methods wherever necessary.